src/editor/locale/lang.fr.js

```diff
@@ -259,7 +259,7 @@ export default {
     jgraduate_ellip: 'Ellip:',
     jgraduate_set_ellip: 'Click to set Ellip',
     jgraduate_angle: 'Angle:',
-    jgraduate_set_angle: 'Click to set Angle',
+    jgraduate_set_angle: 'Cliquez pour définire l\'angle Angle',
```

Comment:
définir

Reply:
Attention: 'angle' is repeated twice.

Reply:
Ah ! Sorry. I fix it.

Reply:
Good, I will merge it as soon as your PR is updated. Thanks again.

Reply:
I fixed these tkings